metadata-ingestion/src/datahub/configuration/_config_enum.py

```diff
@@ -15,11 +17,25 @@ def _generate_next_value_(  # type: ignore
         # From https://stackoverflow.com/a/44785241/5004662.
         return name
 
-    @classmethod
-    def __get_validators__(cls) -> "pydantic.types.CallableGenerator":
-        # We convert the text to uppercase before attempting to match it to an enum value.
-        yield cls.validate
-        yield pydantic.validators.enum_member_validator
+    if PYDANTIC_VERSION_2:
+        # if TYPE_CHECKING:
```

Comment:
Is this intentionally commented out?

Reply:
yes - mainly as a marker for how to type this function in the future